ambry-network/src/main/java/com.github.ambry.network/SSLTransmission.java

```diff
@@ -444,7 +460,7 @@ public int read(ByteBuffer dst) throws IOException {
         }
```

Comment:
Move the log and metric outside of the while loop? `while (dst.remaining() > 0) `

Reply:
The metric in `read()` already covers this case.